run-health-checks.sh

```diff
@@ -93,34 +93,42 @@ if [ -z "$CONF_FILE" ]; then
         conf_name="hb120rs_v3"
     elif echo "$SKU" | grep -q "hb176rs_v4"; then
         conf_name="hb176rs_v4"
-	an_rate=100
+	    an_rate=100
```

Comment:
Indentation off?

Reply:
Yea, it shows different in different terminals/editors. Let me revert as I thought I was actually fixing the indention issue

Reply:
addressed